module5/lessons/Week1/APIReview.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: page
+title: API Review
+---
+
+Before we start in on new topics for Mod5, let's review something we learned in Mod4 - implementing API endpoints!
+
+We will be building on a application based on the database you worked with during intermission.
+
+For this exercise, you can assume that your instructors are the client. Any question that you would want to ask your client should be directed to the instructors!
+
+## Setup
+* Fork and Clone [this starter repo](https://github.com/turingschool-examples/CountryClub)
+* `update-database` <- this will take a while! There is a lot of seed data.
+* Run the application and make sure you are seeing a list of members when you `GET http://localhost:5295/api/members` in postman
+
+## CRUD Members
+
+One endpoint exists to return a list of all members:
+`GET api/members`
+
+<section class='call-to-action' markdown='1'>
+
+Check out a new branch, and create the following endpoints:
+
+* `GET api/members/:id` This should return a single member.
+* `POST api/members` This should create a new member.  You will need to send some information in the request body for this one!
+* `PUT api/members/:id` This should update a member. You will need to send some information in the request body for this one!
+* `DELETE api/members/:id` This should remove a member from the database.
+    * 🌶️optional: you could set the member as 'inactive' to preserve data history.  This will require more than an endpoint 😉
+
+✅Create a Pull Request, and ask one of your cohort-mates to review it before merging.
```

Comment:
This is a small thing, but maybe but a line break at the end of these two deliverables so they actually show up on a new line. Although they might already be doing that on the website, unsure 🤷‍♀️